web-registry/src/components/organisms/BastketDetailsActionsBar/BastketDetailsActionsBar.tsx

```diff
@@ -0,0 +1,54 @@
+import { Box } from '@mui/material';
+
+import ContainedButton from 'web-components/lib/components/buttons/ContainedButton';
+import CurrentCreditsIcon from 'web-components/lib/components/icons/CurrentCreditsIcon';
+import { StickyBar } from 'web-components/lib/components/sticky-bar/StickyBar';
+import InfoTooltipWithIcon from 'web-components/lib/components/tooltip/InfoTooltipWithIcon';
+
+import { Link } from 'components/atoms';
+
+import {
+  BUY_BASKET_TOKENS,
+  BUY_BASKET_TOKENS_TOOLTIP,
+  BUY_BUTTON_LINK,
+} from './BastketDetailsActionsBar.constants';
+
+type Params = {
+  isBuyButtonDisabled?: boolean;
+  onBuyButtonClick?: () => void;
+};
+
+export const BastketDetailsActionsBar = ({
```

Comment:
```suggestion
export const BasketDetailsActionsBar = ({
```
the file names should also be `Basket` and not `Bastket` - I’m not sure we need to change this before merging though

Reply:
Nice catch, will fix this typo.